rest/accounts/instance-get-example-1/instance-get-example-1.7.x.java

```diff
@@ -0,0 +1,20 @@
+// Install the Java helper library from twilio.com/docs/java/install
+import com.twilio.rest.Twilio;
+import com.twilio.rest.fetcher.api.v2010.AccountFetcher;
+import com.twilio.rest.resource.api.v2010.Account;
+
+public class Example {
+  // Find your Account Sid and Token at twilio.com/user/account
+  public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+  public static final String AUTH_TOKEN = "your_auth_token";
+
+  public static void main(String[] args) {
+    Twilio.init(ACCOUNT_SID, AUTH_TOKEN);
+
+    // Get an object from its sid. If you do not have a sid,
+    // check out the list resource examples on this page
+    Account account = new AccountFetcher(ACCOUNT_SID).execute();
```

Comment:
For 2010 resources, we also provide a convenience constructor so that you don't have to pass in the account sid. We will then pull the account sid off of the client. 

`new AccountFetcher().execute();`